### PR TITLE
remove ticket number requirement from pr tempate

### PR DESCRIPTION
**Pull request by AndriusBartulis**
## Changes summary

- Changes the ticket number requirement to be just text rather than a URL as the URL will be included automatically

## Risks

-

## Manual testing done

-

### CODE REVIEW

**Review by timmw — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# [DEV-]()
```
Comment (jam3sn) [inline PULL_REQUEST_TEMPLATE.md:1]: Hmm it's worked for me without linking the ticket ID in the description, see this PR https://github.com/sparklayer-io/spark-api-theme/pull/44

Perhaps we update it to `# DEV-`

Comment (AndriusBartulis) [reply PULL_REQUEST_TEMPLATE.md:1]: Sounds good, lets try it!

Comment (AndriusBartulis) [reply PULL_REQUEST_TEMPLATE.md:1]: @jam3sn hmm tried it here on this ticket and it did not work: https://github.com/sparklayer-io/spark-js/pull/1226

Comment (jam3sn) [reply PULL_REQUEST_TEMPLATE.md:1]: Must be a larger issue with the integration then :/

**Review by jam3sn — APPROVED** [review]

**Review by timmw — APPROVED** [review]

**Pull request merged**